.github/workflows/github-release.yml

```
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2020 CERN.
+#
+# Invenio is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details
+
+name: GitHub Release
+
+on:
+  push:
+    tags:
+      - v*
+
+jobs:
+  Release:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: "GitHub Release"
+
+      - uses: "marvinpinto/action-automatic-releases@latest"
```

I think it will not work with extra dash because it is another step:
```suggestion
        uses: "marvinpinto/action-automatic-releases@latest"
```
Or it will have an empty name and the previous one an empty execution...
I might be wrong...

---

Let's give it a try. This one is very tricky to test...